docs/reference/commandline/dockerd.md

```diff
@@ -396,61 +396,241 @@ Defaults to 20G.
 C:\> dockerd --storage-opt size=40G
 ```
 
-### Docker runtime execution options
+### Runtime options
 
 The Docker daemon relies on a
 [OCI](https://github.com/opencontainers/runtime-spec) compliant runtime
 (invoked via the `containerd` daemon) as its interface to the Linux
 kernel `namespaces`, `cgroups`, and `SELinux`.
 
-By default, the Docker daemon automatically starts `containerd`. If you want to
-control `containerd` startup, manually start `containerd` and pass the path to
-the `containerd` socket using the `--containerd` flag. For example:
+#### Configure container runtimes
+
+By default, the Docker daemon uses runc as a container runtime.
+You can configure the daemon to add additional runtimes.
+
+containerd shims installed on `PATH` can be used directly, without the need
+to edit the daemon's configuration.
```

Comment:
How does one use them directly? The only mention of `docker run --runtime` in this page is in the context of using a _registered_ runtime so users may not make the connection that `docker run --runtime io.containerd.kata.v2` can be used to run containers with Kata once `containerd-shim-kata-v2` is installed on `PATH`, without any further configuration.